model/articles.model.js

```diff
@@ -47,10 +51,22 @@ exports.fetchArticles = (sortBy = "author", order = "ASC") => {
     "created_at",
     "votes",
   ];
+  const topicGreenList = ["mitch", "cats", "paper"];
```

Comment:
We need to query the db to find the valid topics, as new topics could be added to the db which would cause this code to break

Reply:
is that just going to be a SELECT topic FROM topics ?